meta-ivi/recipes-support-ivi/ivi-logging/ivi-logging.inc

```diff
@@ -0,0 +1,23 @@
+DESCRIPTION = "ivi-logging"
+HOMEPAGE = "http://www.pelagicore.com"
```

Comment:
DESCRIPTION ought to be a bit more verbose. 1-2 sentences.

HOMEPAGE should point to a proper software project home page for this component, otherwise it does not serve any purpose IMHO.  It would be better to leave it out.  If you think it's important then I suppose you could put the home page link in the comment header, although it's not customary in bitbake recipes to have a lot of extra info in comments.